include/unicorn/x86.h

```diff
@@ -8,6 +8,15 @@
 extern "C" {
 #endif
 
+//Memory-Management Register fields (idtr, gdtr, ldtr, tr)
+//borrow from SegmentCache in qemu/target-i386/cpu.h
+typedef struct x86_mmr {
+    uint32_t selector;
+    uint64_t base;   /* handle 32 or 64 bit CPUs */
+    uint32_t limit;
+    uint32_t flags;
+} x86_mmr;
+
```

Comment:
in case we just want 1 struct for everything, then we need to document that IDTR & GDTR do not use "selector" & "flags" fields.